audio/softsynth/fluidsynth.cpp

```diff
@@ -179,7 +184,16 @@ int MidiDriver_FluidSynth::open() {
 
 	const char *soundfont = ConfMan.get("soundfont").c_str();
 
+#ifdef IPHONE_OFFICIAL
+	char *soundfont_fullpath[PATH_MAX];
+	const char *document_path = iOS7_getDocumentsDir();
```

Comment:
Looks like `iPHONE_OFFICIAL` is a valid define for pre-iOS7 backend too. Thus, this is probably breaking build for that.